test/e2e/play_kube_test.go

```diff
@@ -5850,4 +5850,31 @@ EXPOSE 2004-2005/tcp`, ALPINE)
 		Expect(session).Should(Exit(0))
 		Expect(session.OutputToString()).To(Equal("true"))
 	})
+
+	It("podman kube play test with valid Umask value", func() {
+		ctrName := "demo-ctr"
+		podName := "demo-pod"
+		outputFile := filepath.Join(podmanTest.TempDir, "pod.yaml")
+
+		session := podmanTest.Podman([]string{"create", "--name", ctrName, "--pod", "new:" + podName, ALPINE})
+		session.WaitWithDefaultTimeout()
+		Expect(session).Should(Exit(0))
+
+		session = podmanTest.Podman([]string{"kube", "generate", "-f", outputFile, podName})
+		session.WaitWithDefaultTimeout()
+		Expect(session).Should(Exit(0))
```

Comment:
this creates a yaml with restart policy always which means the started container will juts restart all time causeing unnecessary load